Fix 'Unknown CMake command "check_symbol_exists".' error
By Sebastien Varrette (UL HPC Team - University of Luxembourg)
diff -Nru arpack-ng-3.7.0.orig/CMakeLists.txt arpack-ng-3.7.0/CMakeLists.txt
--- arpack-ng-3.7.0.orig/CMakeLists.txt	2020-04-07 15:46:29.000000000 +0200
+++ arpack-ng-3.7.0/CMakeLists.txt	2020-04-07 15:47:35.000000000 +0200
@@ -29,6 +29,8 @@
 
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 
+include(CheckSymbolExists)
+
 if (COVERALLS)
     include(Coveralls)
     set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -g -O0 -fprofile-arcs -ftest-coverage")
